docs/architecture/navigation.rst

```diff
@@ -0,0 +1,34 @@
+Navigation
+==========
+
+Saleor gives a possibility to configure storefront navigation. It can be done in dashboard ``Navigation`` tab.
+
+You can add up to 3 levels of menu items inside every menu you create. Each menu item can point to an internal page with Category, Collection or Page, or an external website by passing an extra URL.
+
+
+Managing menu items
+-------------------
+
+To manage menu items, first you must go to dashboard ``Navigation`` tab and choose menu to edit. If you want to manage nested menu items, you can navigate through listed menu items up and down.
+
+To add new menu item, choose ``Add`` button visible above the list of menu items. Then fill up the form and click ``Create``.
+
+To edit menu item, choose ``Edit`` button visible next to a menu item on the list or ``Edit menu item`` button below menu item details, if you're inside menu item details view. Make any changes and click ``Update``.
+
+To remove a menu item, choose ``Remove`` button visible next to a menu item on the list or ``Remove menu item`` button below menu item details, if you're inside menu item details view. This action will remove all descendant items and can't be undone.
```

Comment:
I think information about irreversible action should be at least **bolded**, maybe adding `WARNING` would save someone tears too.

Reply:
Not sure of that. All actions that remove objects from DB are irreversible for now. And we display modal on remove action that lists descendant menu items to remove.